Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -134,31 +145,65 @@ properties.buildReportOrder.each{ buildReportFile ->
 		if(executeRecord.getOutputs().isEmpty() != true) {
 			count += executeRecord.getOutputs().size()
 			executeRecord.getOutputs().each{ output ->
-				def (ds,member) = getDatasetName(output.dataset)
+				def (ds,member) = getContainerAndFile(output.dataset)
 				tempBuildOutputsMap.put(new DeployableArtifact(member, output.deployType) , [ds, buildReport, executeRecord, buildReportRank])
 			}
 		}
 	}
 
+    // store build output information in Hashmap buildOutputsMap to replace potential duplicates
+    // managing TYPE USS_RECORD
+    USSRecords.each{ USSRecord ->
+        ArrayList<ArrayList> outputs = []
+        USSRecord.getAttribute("outputs").split(';').collectEntries { entry ->
```

Comment:
I assume that this is a temporary solution to split it by `;` until the RFE [DBB-I-41](https://ideas.ibm.com/ideas/DBB-I-43) is available. Can we make a note about the RFE in a comment in the code?

Reply:
Yes it is - When the official USS_RECORD is available, we'll support this one.